src/ontology/uberon-edit.obo

```diff
@@ -191,6 +191,7 @@ property_value: doap-SVNRepository https://obo.svn.sourceforge.net/svnroot/obo/u
 property_value: foaf-homepage http://uberon.org xsd:anyURI
 property_value: has_ontology_root_term UBERON:0000104
 property_value: has_ontology_root_term UBERON:0001062
+owl-axioms: Prefix(owl:=<http://www.w3.org/2002/07/owl#>)\nPrefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)\nPrefix(xml:=<http://www.w3.org/XML/1998/namespace>)\nPrefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)\nPrefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)\n\n\nOntology(\nDeclaration(Class(<http://purl.obolibrary.org/obo/UBERON_0006428>))\nDeclaration(Class(<http://purl.obolibrary.org/obo/UBERON_0006983>))\nDeclaration(Class(<http://purl.obolibrary.org/obo/UBERON_0008828>))\nDeclaration(Class(<http://purl.obolibrary.org/obo/UBERON_0009199>))\nDeclaration(Class(<http://purl.obolibrary.org/obo/UBERON_7500110>))\nDeclaration(Class(<http://purl.obolibrary.org/obo/UBERON_7500111>))\nDeclaration(ObjectProperty(<http://purl.obolibrary.org/obo/BFO_0000050>))\nDeclaration(ObjectProperty(<http://purl.obolibrary.org/obo/RO_0002176>))\n\n############################\n#   Classes\n############################\n\n# Class: <http://purl.obolibrary.org/obo/UBERON_7500110> (<http://purl.obolibrary.org/obo/UBERON_7500110>)\n\nSubClassOf(<http://purl.obolibrary.org/obo/UBERON_7500110> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/UBERON_0006983> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/UBERON_7500111>)))\n\n# Class: <http://purl.obolibrary.org/obo/UBERON_7500111> (<http://purl.obolibrary.org/obo/UBERON_7500111>)\n\nSubClassOf(<http://purl.obolibrary.org/obo/UBERON_7500111> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/UBERON_0009199> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002176> <http://purl.obolibrary.org/obo/UBERON_0006428>) ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002176> <http://purl.obolibrary.org/obo/UBERON_0008828>)))\n\n\n)
```

Comment:
Seeing duplicates in the Anonymous Ancestor:
<img width="505" alt="Screenshot 2022-08-12 at 15 51 43" src="https://user-images.githubusercontent.com/95643797/184381458-69bc8755-e6c8-4eaa-ab0c-d1e0a1f57887.png">

Reply:
I think this is the difference in how CARO and UBERON define anatomical entity and anatomical point.

Reply:
@megbalk no need for the intersection here. Just use two subclass axioms: 
- `'anatomical point'`
- `'part of' some 'inter sphenoid suture'`